packages/@lwc/engine-server/src/__tests__/fixtures-renderer.spec.ts

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
Let's reuse the existing test suite for this, it already accepts a `config.json` instead of creating a brand new suite. We can pass for example an `option` object with `syntheticShadow` for example.

Reply:
Ok that was my first implementation, where all the tests are ran with synthetic shadow true and false. I restored it.